src/Nethermind/Nethermind.Network/SessionMonitor.cs

```diff
@@ -60,21 +60,9 @@ private void OnDisconnected(object sender, DisconnectEventArgs e)
             _sessions.TryRemove(session.SessionId, out session);
         }
 
-        private void SendPingMessages()
-        {
-            Task task = Task.Run(SendPingMessagesAsync).ContinueWith(x =>
-            {
-                if (x.IsFaulted && _logger.IsError)
-                {
-                    if (_logger.IsDebug) _logger.Error($"DEBUG/ERROR Error during send ping messages: {x.Exception}");
-                }
-            });
-
-            task.Wait();
-        }
-
         private async Task SendPingMessagesAsync()
         {
+            await Task.Yield();
```

Comment:
Why yield?

Reply:
Was using `Task.Run` previously so maintains that behaviour of queuing to threadpool thread (as above); but removes the `.Wait()` block on the Timer thread